R/margins.R

```diff
@@ -252,9 +261,16 @@ justify_grobs <- function(grobs, x = NULL, y = NULL, hjust = 0.5, vjust = 0.5, d
     return(grobs)
   }
 
+  # adjust hjust and vjust according to internal angle
+  rad <- (int_angle %||% 0) * 2 * pi / 360
```

Comment:
Worth extracting out into a function?

Reply:
In which source file would it live? And do you have a suggestion for a name? `deg_to_rad()`? `convert_to_rad()`? `to_rad()`?

Reply:
Sorry, I meant the whole hjust/vjust angle adjustment

Reply:
Ah, yes, I had thought about that myself at some point. Something like `rotate_just()` that takes `hjust` and `vjust` and returns rotated versions of them. Will do. Still the question is where does this live? In `margins.R`?

Reply:
Yeah, I guess. At some point we probably need to arrange these functions to all be in the same place.